src/clj/game/cards/assets.clj

```diff
@@ -1483,6 +1483,16 @@
    "Turtlebacks"
    {:events {:server-created {:msg "gain 1 [Credits]" :effect (effect (gain :credit 1))}}}
 
+   "Urban Renewal"
+   {:effect (effect (add-counter card :power 3))
+    :derezzed-events {:runner-turn-ends corp-rez-toast}
+    :events {:corp-turn-begins
+             {:effect (req (add-counter state side card :power -1)
+                           (when (<= (get-in card [:counter :power]) 1)
```

Comment:
Since this ability needs to be `:delayed-completion`, you'll need to trigger `effect-completed` if this condition is false.